docs/bots-guide.md

```diff
@@ -346,6 +346,62 @@ client.update_message(dict(
 * My bot works only on some streams.
     * Subscribe your bot to other streams, as described [here](#how-to-run-a-bot).
 
+## Flask Server
+It is a simple python flask server which can be used to interact with bots through making use of
+outgoing webhook services. It makes it really convenient for users to run multiple bots. Traditionally,
+a person would be required to download zuliprc of each bot and run each of them separately. With Flask
+server, he would be able to download a single configuration file for all bots i.e. `flaskbotrc` and
+start flask server which would enable interacting with all the bots.
+
+It's location is `api/flask_bot_server.py`. Following are the steps to set it up and use it:
+
+1.  Make sure that bot with whom you want to interact has been setup and is active (lets call it bot1).
+
+2.  Download the `flaskbotrc` through the download icon below `Add a new bot` form. It contains the
+    configuration details for all the active bots. It's structure is very similar to that of zuliprc.
+
+3.  Change directory to `api/` and run the flask server by passing the `flaskbotrc` to it.
+    The command format is:
+    ```
+    cd api
+    python flask_bot_server.py <path_to_flaskbotrc>
+    ```
+    Sample command:
+    ```
+    python flask_bot_server.py ~/flaskbotrc
+    ```
+    Make sure that the flask server is being run inside the vagrant machine.
+
+4.  Now setup the outgoing webhook service which will interact with the flask server. For this create an
+    `outgoing webhook` type bot (lets call this bot2) and create a service with base url of the form:
+    ```
+    http://127.0.0.1:5002/bots/<bot_name>
+    ```
+    The `bot name` here refers to the name of the bot with whom you want to interact. It can be obtained by
+    removing `-bot@*.*` from the bot email. Example: the bot name of bot with email `followup-bot@zulip.com`
+    is `followup`.
+
+5.  Now with everything setup, you can start interacting with the bot (bot1 here) by triggering the outgoing
+    webhook service, through @-mentioning or sending a private message to bot2. The content of the above message
+    should be text you want to send to bot1.
+
+    Please note that in order to @-mention trigger a bot on a stream, the bot needs to be subscribed to it.
+
+## Followup bot example
+
+1.  First setup the followup bot by creating a new bot with username `followup`.
+    Create a new followup stream and subscribe the bot to the stream where it will be triggered.
```

Comment:
Does the bot need to be subscribed to BOTH the followup stream as well as the stream where it will be triggered?